src/oci.c

```diff
@@ -732,6 +732,11 @@ cc_oci_start (struct cc_oci_config *config,
 			ret = false;
 			goto out;
 		}
+	} else if (cc_pod_is_sandbox(config)) {
+		cc_proxy_hyper_new_pod_container(config,
+						config->optarg_container_id,
+						config->optarg_container_id,
+						"rootfs", config->optarg_container_id);
 	} else if (config->pod && ! config->pod->sandbox) {
```

Comment:
Just for more clarity, maybe use `else if (! cc_pod_is_sandbox(config))` here, instead of else if `(config->pod && ! config->pod->sandbox)`.